pkg/controller-gen/main.go

```diff
@@ -117,6 +123,10 @@ func Run(opts cgargs.Options) {
 		logrus.Fatalf("informers failed: %v", err)
 	}
 
+	if err := generateOpenAPI(openAPIGroups, customArgs); err != nil {
+		logrus.Fatalf("openapi docs: %v", err)
```

Comment:
Shouldn't the error message be `"openapi failed: %v", err` ?

Also, unless I'm missing something, all the `"...%v", err)` instances in the `Run` function here should be using `%w` instead. This should probably be a separate TechDebt issue, but you can start at this line.

Reply:
Ok, agreed! Do you want me to  change others as well ?

Reply:
Ok, agreed!

Reply:
Sure. I don't know if there are tests hitting that code in `rancher/rancher`, but it's standard to use `%w` on errors, so why not.

Reply:
logrus.Fatalf doesn't support %w, so I had to revert it to %v again.